crates/re_log_types/src/time_point/mod.rs

```diff
@@ -87,6 +88,22 @@ impl TimePoint {
         }
         self
     }
+
+    /// Returns an accurate-enough approximation of the heap allocated size of `self`.
+    #[inline]
+    pub fn size_bytes(&self) -> u64 {
```

Comment:
If you would `impl SizeBytesExt` instead here and elsewhere, we would get the benefit of only having to document the semantics of `size_bytes` once